### PR TITLE
Added logging start time and request duration

### PR DESCRIPTION
**Pull request by DFSko**
Added logging start time and request duration

resolves happy-travel/agent-app-project#466

### CODE REVIEW

File: HappyTravel.HttpRequestAuditLogger/Models/HttpRequestAuditLogEntry.cs [hunk 1]
```diff
@@ -13,5 +14,7 @@ public record HttpRequestAuditLogEntry
         public int StatusCode { get; init; }
         public string? ReferenceCode { get; init; }
         public string? Error { get; init; }
+        public DateTime StartTime { get; init; }
+        public double Duration { get; set; }
```
Comment (mingazhev) [inline HappyTravel.HttpRequestAuditLogger/Models/HttpRequestAuditLogEntry.cs:18]: DurationMilliseconds maybe?

File: HappyTravel.HttpRequestAuditLogger/Models/HttpRequestAuditLogEntry.cs [hunk 2]
```diff
@@ -13,5 +14,7 @@ public record HttpRequestAuditLogEntry
         public int StatusCode { get; init; }
         public string? ReferenceCode { get; init; }
         public string? Error { get; init; }
+        public DateTime StartTime { get; init; }
```
Comment (mingazhev) [inline HappyTravel.HttpRequestAuditLogger/Models/HttpRequestAuditLogEntry.cs:17]: StartedAt maybe? Looks like not only the time is actual there

File: HappyTravel.HttpRequestAuditLogger/Models/HttpRequestAuditLogEntry.cs (hunk 1 above)
Comment (DFSko) [reply HappyTravel.HttpRequestAuditLogger/Models/HttpRequestAuditLogEntry.cs]: Fixed

File: HappyTravel.HttpRequestAuditLogger/Models/HttpRequestAuditLogEntry.cs (hunk 2 above)
Comment (DFSko) [reply HappyTravel.HttpRequestAuditLogger/Models/HttpRequestAuditLogEntry.cs]: Fixed

**Review by mingazhev — APPROVED** [review]

**Pull request merged**